frontend/internal-packages/db/supabase/migrations/20250814082520_create_public_share_realtime.sql

```diff
@@ -0,0 +1,118 @@
+-- Table to track which design sessions are publicly shared
+create table if not exists "public"."public_share_settings" (
+    "design_session_id" "uuid" not null,
+    "created_at" timestamp with time zone default "now"() not null
+);
```

Comment:
table for visibility toggle.


PoC (not yet pushed)

https://github.com/user-attachments/assets/5bd8fd57-8b73-4ee0-bf04-2c025071656e


1. Change the value `Private` -> `Anyone with the link`...
     - INSERT to this table.
2. Change the value `Anyone with the link` -> `Private` ...
     - DELETE from this table.